response_operations_ui/controllers/case_controller.py

```diff
@@ -26,3 +26,59 @@ def update_case_group_statuses(short_name, period, ru_ref, event):
     response = requests.post(url, json={'event': event})
     if response.status_code != 200:
         raise ApiError(response)
+
+    logger.debug('Successfully updated case group status', short_name=short_name, period=period, ru_ref=ru_ref)
+
+
+def get_available_case_group_statuses_direct(collection_exercise_id, ru_ref):
```

Comment:
🎨  I'm not sure how useful `_direct` is here. Is this so we can eventually deprecate the existing functionality (backstage)? It would be better if this PR actually replaced the calls to backstage rather than adding more functions that do the same thing.

Reply:
The direct name is just temporary
This PR removes the calls required for building the reporting unit page and the original function `get_available_case_group_statuses` is a call to backstage used by a different part of the application so it hasn't been removed yet